lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -3018,9 +3018,9 @@ namespace Js
         return this->GetLineCharOffsetFromStartChar(startCharOfStatement, _line, _charOffset, canAllocateLineCache);
     }
 
-    bool FunctionBody::GetLineCharOffsetFromStartChar(int startCharOfStatement, ULONG* _line, LONG* _charOffset, bool canAllocateLineCache /*= true*/)
+    bool FunctionBody::GetLineCharOffsetFromStartChar(int startCharOfStatement, ULONG* _line, LONG* _charOffset, bool canAllocateLineCache /*= true*/, bool canDumpLibraryCode /*= false*/)
     {
-        Assert(!this->GetUtf8SourceInfo()->GetIsLibraryCode());
+        Assert(canDumpLibraryCode || !this->GetUtf8SourceInfo()->GetIsLibraryCode());
```

Comment:
Thought exercise: in release builds, Asserts are compiled out and won't hit. We want to ensure we still get the intended (and reasonable) behavior even if the Assert didn't cause execution to stop.